extra_settings/models.py

```diff
@@ -208,7 +216,17 @@ def __init__(self, *args, **kwargs):
             self.value = value
         self.name_initial = self.name
 
+    def validate(self):
+        if not self.validator:
+            return
+        module_name, validator_func_name = str(self.validator).rsplit(".", 1)
+        module = importlib.import_module(module_name)
+        validator = getattr(module, validator_func_name)
+        if not validator(self.value):
+            raise ValueError("{} could not be validated by {} validator.".format(self.value, validator_func_name))
+
```

Comment:
I recently had the same need and I ended up writing this `import_function` method that you could add to `utils` and use for loading the validator func.


```python
from django.utils.module_loading import import_string


def import_function(path):
    try:
        func = import_string(path)
        if hasattr(func, "__call__"):
            return func
        return None
    except ImportError:
        return None
```